schedule.html

```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      W3C Workshop on Web &amp; Virtual Reality
+    </title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <p>
+      <a href="https://www.w3.org/"><img alt="W3C" src=
+      "https://www.w3.org/Icons/w3c_home" height="48" width="72"></a>
+    </p>
+    <h1 id="top">
+      W3C Workshop on Web &amp; Virtual Reality
+    </h1>
+    <p>
+      October 19-20, 2016; San Jose, CA, USA
+    </p>
+    <ul class="menu">
```

Comment:
I understand splitting content into separate sections, but this will be difficult to manage as we add more pages and have to remember to update the same markup and text on every page.

I have a local branch that consolidates this all; it's not super crucial, I'll make a PR when I have time. it basically consolidates all the info. onto the `index.html` page but shows only the relevant content when the hash is selected (e.g., https://w3c.github.io/vr-workshop/#schedule would hide the current Home info. [e.g., intro/goals/etc.] and instead toggle the visibility of the Schedule info [i.e., this page]). and then I have redirects for all the pages, so https://w3c.github.io/vr-workshop/schedule (or https://w3c.github.io/vr-workshop/schedule.html) becomes https://w3c.github.io/vr-workshop/#schedule.